css/layouts/search.scss

```diff
@@ -0,0 +1,63 @@
+// styling for search popover from header
+
+.header-right {
+    .unified-search__input-wrapper label {
+        display: none;
+    }
+    
+    .unified-search__input-wrapper ~ .empty-content {
+        display: none;
+    }
+    
+    .empty-content {
+        &__icon > .material-design-icon.magnify-icon{
+            display: none;
+        }
+    }
+
+    .header-menu {
+        &__carret {
+            display: none;
+        }
+    }
+}
+
+#unified-search__input {
+    border: 2px solid black;
+    border-radius: 5px;
+    padding-top: 1.5rem;
+    padding-bottom: 1.5rem;
+    padding-left: 1rem;
+}
+
+
+.unified-search__result-icon.unified-search__result-icon--no-preview {
+    &.icon-folder {
+        background-size: 64px;
+        background-image: url('../img/filetypes/folder.svg');
```

Comment:
variable for icon should be used

Reply:
Hello Aleksei, which variable should we use for this icon and where do I find it/ declare it?

Reply:
in src/icons.mjs you can find an icon generation script. To run it just do 'npm run sass:icons'. The generated file is located in dist/icons.css, you can see all the available icons there. For folder.svg we have --icon-folder-dark

Reply:
Used --icon-mime-folder-dark as discussed, thank you